CoreStubs/ArchiveControl/module.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+class ArchiveControl extends IPSModule
```

Comment:
Die fehlenden Funktionen bitte mit "throw Not implemented" einbinden, siehe diverse andere Klassen